### PR TITLE
Improvement/#4 client servings

### PR DESCRIPTION
**Pull request by GalochkinKirov**

### CODE REVIEW

**Comment by OAGr** [comment]
Nice work so far. 
Here's the corresponding url:
https://prediction-backend-stagi-pr-39.herokuapp.com

I've noticed that corresponding urls are broken. So /login doesn't work.

File: client/package.json [hunk 1]
```diff
@@ -56,10 +57,9 @@
     "react-icons-kit": "^1.2.1",
     "reason-apollo": "^0.13.0",
     "reason-react": ">=0.4.0",
+    "static-server": "^2.2.1",
     "subscriptions-transport-ws": "^0.9.14",
-    "victory": "^32.0.0"
-  },
-  "devDependencies": {
+    "victory": "^32.0.0",
```
Comment (OAGr) [inline client/package.json]: Why is this a devDevependency?

File: server/src/index.js [hunk 2]
```diff
@@ -52,10 +53,15 @@ const server = new ApolloServer({
 const app = express();
 const cors = require("cors");
 app.use(cors());
+app.get(/^((?!graphql).)*$/,
```
Comment (OAGr) [inline server/src/index.js]: Can you add a comment to explain this a bit?

File: client/package.json (hunk 1 above)
Comment (Wirnex) [reply client/package.json]: It's not, it's the **dependencies**

Comment (OAGr) [reply client/package.json]: Oh sorry, I think I misread the commit diff.

File: server/src/index.js (hunk 2 above)
Comment (GalochkinKirov) [reply server/src/index.js]: Yes, sure, did

**Comment by OAGr** [comment]
Nice work on this!

**Pull request merged**